source/performance/F0.CodeAnalysis.Benchmarking/CodeAnalysis/AnalyzerBenchmark.cs

```diff
@@ -59,7 +60,10 @@ protected static Document CreateDocument(SourceText code, LanguageVersion langua
 
 		var solution = CreateSolution(code, languageVersion, projectId, documentId);
 
-		return solution.GetDocument(documentId);
+		var document = solution.GetDocument(documentId);
+
+		Debug.Assert(document is not null, $"{nameof(DocumentId)} {documentId} is not an ID of a document that is part of this solution.");
```

Comment:
We're getting some new _nullable warnings_ due to multi-targeting _.NET 7.0_ in two projects that previously targeted _.NET Standard 2.0_ only.
```diff
-<TargetFramework>netstandard2.0</TargetFramework>
+<TargetFrameworks>net7.0;netstandard2.0</TargetFrameworks>
```